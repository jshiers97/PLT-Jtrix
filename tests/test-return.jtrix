int ret5(){
return 5;
}

bool retT(){
return true;
}

string rets(){
string s;
s = "hello";
return s;
}

int testcast(){
int i;
float j;
j = 187.0;
i = f_to_int(j);
return i;
}

int main(){
string s;
int i;

i = testcast();
print(testcast());
s = rets();
println(s);
print(ret5());
printb(retT());
println(rets());
return 0;
}
