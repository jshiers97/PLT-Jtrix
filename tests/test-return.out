187
hello
5
1
hello
